app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -1,6 +1,8 @@
 package com.zulip.android;
 
 import java.sql.SQLException;
+import java.util.Map;
+import java.util.Queue;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentLinkedQueue;
```

Comment:
@niftynei No they are not eligible for removal because they are still used in this class.